### DISCUSSION TITLE
des `dagster._core.errors.DagsterExecutionInterruptedError` mean the process is cancelled or interrupted by the user or something outside of dagster?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18431] [Q&A]
Heyo, is there a specific error for Dagster when a user terminates a run as opposed to it being stop for other reasons? I have an asset that runs a job on another server that I want to be able to cancel by hitting "Terminate" myself on the UI, but I don't want other interruptions to cause Dagster to try and cancel the execution. I tried `DagsterExecutionInterruptedError` but that's too broad

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1695314277826659?thread_ts=1695314277.826659&cid=C01U954MEER).*

**Comment by gibsondan** [comment]
Covered by https://github.com/dagster-io/dagster/discussions/12943
